src/test/java/org/ice4j/ice/NetworkUtilsTest.java

```diff
@@ -42,6 +76,12 @@ public void testIpv6StringToBytes()
         assertNotNull(addr);
         assertEquals(18, addr[15]);
 
+        assertNull(NetworkUtils.strToIPv6(""));
```

Comment:
Can we add some assertions for IPv6 strings that are _"almost"_ correct, but not quite? For example: 

`684D:1111:222:3333:4444:5555:6` should be an invalid IPv6.

Reply:
Yes, thanks. More lines covered with this.